widget_bandsplot/schemas/bands.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$id": "https://raw.githubusercontent.com/osscar-org/widget-bandsplot/main/widget_bandsplot/schemas/bands.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "JSON Schema for a bandstructure entry",
+    "type": "object",
+    "required": ["path", "paths", "fermi_level"],
```

Comment:
I think it may be more proper to be called "paths" -> "bands"? The issue is that the key name is get from AiiDA bandsData node exported I guess, okay to keep it here and we change it in the next iteration.